### ISSUE TITLE
Can the DigiApp handle s.l. / s.s. ?

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Mass-Digitizer#499]
Some folders at the Herbarium have s.l. / s.s. (sensu lato/ sensu stricto - in a broad / strict sense) added to the Taxon name. Should the digitizers add this when digitizing? Can the DigiApp and the GREL script handle this? Do we want this information in Specify, and if yes, do we have to implement this ourselves or is it already possible?

Example: "Angelica archangelica L. s.l."

Necessary steps:
- Discuss with @PipBrewer and collections managers if we want to add this into Specify.
- If we want to, talk to Fedor about how we should do this.

**Comment by PipBrewer** [comment]
I reckon that would be treated as a suffix to the taxon name in the specimen record (not the taxon record). But would want to check with Jens S. 

**Action:** Pip to check with Jens S.

**Comment by PipBrewer** [comment]
@FedorSteeman @AstridBVW The ss/sl should be added to the specimen record (to the addendum) in Specify. Do you have any suggestions about how we would deal with this in the DigiApp?

**Comment by RebekkaML** [comment]
Discussed this issue with Fedor Steemann and Zsuzsanna at the Data Issue Meeting 13.06.2024. The GREL script for imports into Specify needs to be reviewed and we need to determine how exactly it handles these instances and if it needs to be adjusted.

**Comment by PipBrewer** [comment]
Did we not agree that @AstridBVW or @FedorSteeman would look into this further?

**Comment by RebekkaML** [comment]
Yes, that was what we decided on the data meeting (13.06.2024). I could have made the comment more specific.

**Comment by PipBrewer** [comment]
Let's treat this on a case-by-case basis. For this specific example, let's ask Jens S whether that sl. in this case should be part of the taxon record or the determination (as a suffix).

**Comment by FedorSteeman** [comment]
Yeah, this still needs to be looked into.

**Comment by beckerah** [comment]
Pip has asked the collection managers if we can add this data to the "addendum" field in Specify. If they approve, I would suggest that the digitizers input this information as a note in the DigiApp (preferably spelling out sensu lato or sensu stricto). Can we get a barcode for these like we did for "loco ignoto vel cult."?

If that's all satisfactory, I'll update the GREL script to create this column in the exports and move the data to a new "addendum" column.

**Comment by beckerah** [comment]
I talked to Jens about this in person and he approves our suggestion to add these notes to the addendum field in Specify. 

@PipBrewer @RebekkaML @chelseagraham What do you all think of my idea (in the comment just above) of having the digitizers input this in the notes field, possibly with a barcode? That would make it really easy for me to automate separating this data into an 'addendum' column for the Specify import. I currently only have one example of this on a DigiApp export and that's exactly how it was handled: spelled out "sensu lato" in the notes field.

**Comment by chelseagraham** [comment]
> I talked to Jens about this in person and he approves our suggestion to add these notes to the addendum field in Specify.
> 
> @PipBrewer @RebekkaML @chelseagraham What do you all think of my idea (in the comment just above) of having the digitizers input this in the notes field, possibly with a barcode? That would make it really easy for me to automate separating this data into an 'addendum' column for the Specify import. I currently only have one example of this on a DigiApp export and that's exactly how it was handled: spelled out "sensu lato" in the notes field.

I agree with your suggestion that digitizers create controlled vocabulary 'sensu lato' and 'sensu stricto' data matrices to be left at barcoding/DigiApp workstations for scanning into the notes field. Perfect, @beckerah :D

**Comment by RebekkaML** [comment]
I will create barcodes for this and place them at the Herbarium.

**Comment by beckerah** [comment]
I've fixed this in the grel script (#540) and just need someone to add the data matrices to the workstations. @RebekkaML or @chelseagraham - I'm not sure who handles that normally but maybe one of you can take care of it?

**Comment by RebekkaML** [comment]
> I've fixed this in the grel script (#540) and just need someone to add the data matrices to the workstations. @RebekkaML or @chelseagraham - I'm not sure who handles that normally but maybe one of you can take care of it?

I will take care of that.

**Comment by RebekkaML** [comment]
I have created datamatrices and placed them at the workstations.

**Comment by beckerah** [comment]
Thanks, Rebekka! This issue is officially closed. :)
